addons/calendar/models/calendar_event.py

```diff
@@ -823,7 +827,7 @@ def _sync_activities(self, fields):
                 activity_values = {}
                 if 'name' in fields:
                     activity_values['summary'] = event.name
-                if 'description' in fields:
+                if 'description' in fields and event.description:
```

Comment:
there is no indication that this is a reported error (no opw). looks good. functionally I'm not sure if we want to remove the note (do `= event.description and tools.plaintext2html(event.description)`) or do what is done here

Reply:
It's just that is you create/edit an event, it create a activity for this event, but if there is no desc, the description is 'False' 
![](https://cdn.discordapp.com/attachments/896052808023933009/928580899828822097/unknown.png)

Reply:
+ As there was no opw, i solved it in another commit